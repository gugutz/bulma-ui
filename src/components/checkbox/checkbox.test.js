import React from 'react'
import renderer from 'react-test-renderer'
import Checkbox from './checkbox'

describe('<Checkbox /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Checkbox>Remember me</Checkbox>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support disabled prop', () => {
    const tree = renderer
      .create(<Checkbox disabled>Remember me</Checkbox>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
