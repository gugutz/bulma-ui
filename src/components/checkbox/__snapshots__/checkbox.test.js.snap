// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> component should render correctly 1`] = `
<label
  className="checkbox"
  disabled={false}
>
  <input
    disabled={false}
    type="checkbox"
  />
  Remember me
</label>
`;

exports[`<Checkbox /> component should support disabled prop 1`] = `
<label
  className="checkbox"
  disabled={true}
>
  <input
    disabled={true}
    type="checkbox"
  />
  Remember me
</label>
`;
